base/doc/clsguide.tex

```diff
@@ -977,6 +977,30 @@ \subsection{Passing options around}
 handler, but this handler is not used for |landscape| because that option is
 explicitly declared.
 
+\subsection{Testing dates of other code}
+
+\begin{decl}
+  |\IfPackageAtLeast(TF)| \arg{package} \arg{date} \arg{true code} \arg{false code}\\
+  |\IfClassAtLeast(TF)| \arg{class} \arg{date} \arg{true code} \arg{false code}\\
+  |\IfFileAtLeast(TF)| \arg{file} \arg{date} \arg{true code} \arg{false code}
```

Comment:
I think these parentheses can be dropped.
- Existence of no-parentheses forms (at least one occurrence for each command) in PDF help when searching.
- `\arg{true code} \arg{false code}` corresponds to `TF` only.

Suppose there will be more `clsguide` subsections describing `\If...TF` conditionals, a general mention of the single-branch forms could be added at the first of those subsections.

```suggestion
  |\IfPackageAtLeastTF| \arg{package} \arg{date} \arg{true code} \arg{false code}\\
  |\IfClassAtLeastTF| \arg{class} \arg{date} \arg{true code} \arg{false code}\\
  |\IfFileAtLeastTF| \arg{file} \arg{date} \arg{true code} \arg{false code}
```

Reply:
We use the `(TF)` convention in `expl3` and I felt it was clearer - but let's see what others feel.